ui/component/fileDetails/view.jsx

```diff
@@ -42,73 +43,78 @@ class FileDetails extends PureComponent<Props> {
 
     return (
       <Fragment>
-        <Expandable>
-          {description && (
-            <div className="media__info-text">
-              <MarkdownPreview content={description} />
-            </div>
-          )}
-          <ClaimTags uri={uri} type="large" />
-          <table className="table table--condensed table--fixed table--file-details">
-            <tbody>
-              <tr>
-                <td> {__('Content Type')}</td>
-                <td>{mediaType}</td>
-              </tr>
-              {fileSize && (
-                <tr>
-                  <td> {__('File Size')}</td>
-                  <td>{fileSize}</td>
-                </tr>
+        <Card
+          title={__('Details')}
+          body={
+            <Expandable>
```

Comment:
Expandable should become a card option. As should @jessopb's newest collapsible elements (they're collapsible cards).